File: src/agedBrieItem.py
from src.normal_item import *

class AgedBrie(NormalItem):
    
    def __init__(self, name, sell_in, quality):
        Item.__init__(self, name, sell_in, quality)
        
    def setQuality(self, value):
        
        NormalItem.setQuality(self, value)
        
    def updateQuality(self):
        if self.sell_in > 0:
            self.setQuality(1)
        else:
            self.setQuality(2)
        self.setSell_in()

File: gildedrose/src/main/java/edu/craptocraft/gildedrose/domain/AgedBrieItem.java
package edu.craptocraft.gildedrose.domain;

public class AgedBrieItem extends NormalItem{

    public AgedBrieItem(String name, int sell_in, int quality){
        super(name, sell_in, quality);
    }

    @Override
    public void updateQuality(){
        if(getSell_in() > 0){
            computeQuality(1);
        }else{
            computeQuality(2);
        }
        setSell_in();
    }
}
